### PR TITLE
Update deps 4 5 1

### PR DESCRIPTION
**Pull request by alikins**
#### What is this PR doing:

Update deps

Issue: AAH-####

#### Notes: 

**PR Author**: Add a QE reviewer ([exceptions](https://docs.engineering.redhat.com/display/AUTOHUB/Other+Team+Processes#OtherTeamProcesses-galaxy_ngrepo)); 
**Reviewers**: look for sound code, no [code smells](https://www.codegrip.tech/productivity/everything-you-need-to-know-about-code-smells/), docs & test coverage
**Merger**: When merging, include the Jira issue link in the squashed commit

### CODE REVIEW

File: .github/template_gitref [hunk 1]
```diff
@@ -1 +1 @@
-2021.08.26-106-g556224f
+2021.08.26-159-ga96338a
```
Comment (awcrosby) [inline .github/template_gitref:1]: I would not expect this to change if the plugin_template was checked out to `556224f` when the script was run

Comment (alikins) [reply .github/template_gitref:1]: doh, yeah, that pulled in master plugin_template.  Fixing.

File: requirements/requirements.standalone.txt [hunk 2]
```diff
@@ -279,7 +275,7 @@ pyyaml==5.4.1
     #   pulp-ansible
     #   pulpcore
     #   tablib
-redis==4.2.2
+redis==3.4.1
```
Comment (awcrosby) [inline requirements/requirements.standalone.txt:278]: i feel like we dont normally see these dep versions go backwards, might be worth looking at what pulpcore requires (maybe pulpcore started pinning it between 3.18.3 and 3.18.6)

File: template_config.yml [hunk 3]
```diff
@@ -5,15 +5,15 @@
...
   name: pulp_ansible
   org: pulp
 - bindings: true
-  branch: 2.10.2
+  branch: 2.10.5
   name: pulp_container
   org: pulp
 - bindings: false
-  branch: v0.4.2
+  branch: v0.4.5
```
Comment (awcrosby) [inline template_config.yml:16]: You also need to update `pulpcore_branch: 3.18.1` below in this file

Comment (alikins) [reply template_config.yml:16]: Updated.

File: requirements/requirements.standalone.txt (hunk 2 above)
Comment (alikins) [reply requirements/requirements.standalone.txt:278]: Hmm, that is indeed weird. Investigating.

Comment (alikins) [reply requirements/requirements.standalone.txt:278]: Looks like https://github.com/pulp/pulpcore/commit/157842021ea51613f052276caaef26ecabf60bda  changed the pulpcore redis dep from "redis>=3.4.0"  to "redis~=3.4.0"

Hopefully going backwards on redis doesn't break anything.

**Review by newswangerd — APPROVED** [review]
This looks good. It's weird that flake8 is failing though. Could you fix that?

**Comment by alikins** [comment]
> This looks good. It's weird that flake8 is failing though. Could you fix that?

Yup, just updated.

**Review by awcrosby — APPROVED** [review]

**Review by newswangerd — APPROVED** [review]

**Pull request merged**